zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/ElasticsearchStorage.java

```diff
@@ -207,6 +220,22 @@ public final Builder dateSeparator(char dateSeparator) {
      */
     public abstract Builder indexReplicas(int indexReplicas);
 
+    /**
+     * Sets the level of logging for HTTP requests made by the Elasticsearch client. If not set,
+     * logging will be disabled.
+     *
+     * <p>This is only for use from tests and ZipkinElasticsearchStorageConfiguration.
+     */
+    public abstract Builder httpLogging(HttpLoggingLevel httpLoggingLevel);
+
+    /**
+     * Sets the {@link Tracing} to use to trace HTTP requests made by the Elasticsearch client. If
+     * not set, tracing will be disabled.
+     *
+     * <p>This is only for use from tests and ZipkinElasticsearchStorageConfiguration.
+     */
```

Comment:
hmm currently our only api exposure to brave is in zipkin-server. is it possible to keep it there? none of the other libraries have a direct dep on something besides what they use directly